torchci/components/JobAnnotationToggle.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { JobData } from "../lib/types";
+import { ToggleButtonGroup, ToggleButton } from "@mui/material";
+
+type JobAnnotation =
+  | "infra_flake"
+  | "time_out"
+  | "SEV"
+  | "broken_trunk"
+  | "test_flake"
+  | "null";
```

Comment:
For the categories, can we use:
- Broken Trunk
- Test Flakines
- Infra Broken
- Infra Flakiness
- Network Flakiness
- Github/Circle CI Outage
- Other